app/components/AddRuleModal.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { Modal } from "./ModalWrapper";
+export const AddRuleModal: React.SFC<{}> = props => {
+  return (
+    <Modal show={props.showModal} handleClose={props.handleClose}>
+      <label htmlFor="modal-url">URL</label>
+      <input
+        className="form-control"
+        type="text"
+        name="modal-url"
```

Comment:
name like `modal-url` can be better. no need to have modal in context for the name of this input. 
better will be `request_url`.

avoid `-` (hypen) for names.